openfisca_aotearoa/tests/income_tax/family_scheme/family_tax_credit.yaml

```diff
@@ -26,4 +26,47 @@
     - id: "Whare"
       others: ["Papatūānuku", "Rakinui", "Tāne", "Rongo", "Ikatere"]
   output_variables:
-    family_scheme__qualifies_for_family_tax_credit: [1, 0, 0, 0, 0]
\ No newline at end of file
+    family_scheme__qualifies_for_family_tax_credit: [1, 0, 0, 0, 0]
+
+- name: Family tax credit - too much income
+  period: 2018-08
+  absolute_error_margin: 0
+  persons:
+    - id: "Papatūānuku"
+      date_of_birth: 1999-01-01
+      family_scheme__assessable_income:
+        2018: 42700
+  families:
+    - id: "Whanau"
+      principal_caregiver: "Papatūānuku"
+  titled_properties:
+    - id: "Whare"
+      others: ["Papatūānuku"]
+  output_variables:
+    family_scheme__qualifies_for_family_tax_credit:
+      2018-08:
+        - false
+
+- name: Family tax credit - too much income
+  period: 2018-08
+  absolute_error_margin: 0
+  persons:
+    - id: "Papatūānuku"
+      date_of_birth: 1999-01-01
+      family_scheme__assessable_income:
+        2018: 14233
+    - id: "Rakinui"
+      date_of_birth: 1996-01-01
+      family_scheme__assessable_income:
+        2018: 28470
+  families:
+    - id: "Whanau"
+      principal_caregiver: "Papatūānuku"
+      partners: ["Rakinui"]
+  titled_properties:
+    - id: "Whare"
+      others: ["Papatūānuku", "Rakinui"]
+  output_variables:
+    family_scheme__qualifies_for_family_tax_credit:
+      2018-08:
+        - false
```

Comment:
A test with family_scheme__assessable_income that passes the family_scheme__qualifies_for_family_tax_credit is probably a good idea also.

Reply:
There's already a passing test in the file. I added the test for when they're not eligible

Reply:
I'm not sure i understand your comment. You want a test where we pass in a value for `family_scheme__qualifies_for_family_tax_credit`?

Reply:
My bad, I didn't realise the first test was passing in `family_scheme__assessable_income` values, I will mark this resolved.